src/page/personal-information.page.ts

```diff
@@ -0,0 +1,98 @@
+
+  constructor() {
+    this.firstNameInput = element(by.name('firstname'));
+    this.lastNameInput = element(by.name('lastname'));
+    this.sexOptions = element.all(by.name('sex'));
+    this.yearsOfExperienceOptions = element.all(by.name('exp'));
+    this.dateInput = this.selectInput('Date:');
+    this.professionOptions = element.all(by.name('profession'));
+    this.seleniumTools = element.all(by.name('tool'));
+    this.continents = element(by.name('continents'));
+    this.seleniumCommands = element(by.name('selenium_commands'));
+    this.button = element(by.name('submit'));
+  }
+
+  private async selectRadioButton (
+    desiredOption: string, options: ElementArrayFinder): Promise<void> {
+    await options.filter(async (elem) => {
+      const value = await elem.getAttribute('value');
+      return value === desiredOption;
+    }).first().click();
+  }
+
+  private async selectAllCheckBoxs (
+    desiredOptions: string[], options: ElementArrayFinder): Promise<void> {
+    const optionsToSelect: ElementFinder[] = await options.filter(async (elem) => {
+      const value = await elem.getAttribute('value');
+      return desiredOptions.includes(value);
+    });
+    optionsToSelect.forEach(async (elem) => {
+      await elem.click();
+    });
+  }
+
+  private async selectContinent(desiredContinent: string): Promise<void> {
+    const selectedContinent = await this.continents.all(by.tagName('option')).filter(
+      async (elem) => {
+      const  continent = await elem.getText();
+      return continent === desiredContinent;
+    }).first();
+    await selectedContinent.click();
+  }
+
+  private async selectSeleniumComands(desiredComands: string[]): Promise<void> {
+    const selectedCommands: ElementFinder[] = await this.seleniumCommands.all(
+      by.tagName('option')).filter(async (elem) => {
+        const command = await elem.getText();
+        return desiredComands.includes(command);
+      });
+    selectedCommands.forEach(async (elem) => {
+      await elem.click();
+    });
+  }
+
+  public async fillForm(fillValues): Promise<void> {
+    this.dateInput;
+    await this.firstNameInput.sendKeys(fillValues.firstName);
+    await this.lastNameInput.sendKeys(fillValues.lastName);
+    await this.selectRadioButton(fillValues.sex, this.sexOptions);
+    await this.selectRadioButton(fillValues.experience.toString(), this.yearsOfExperienceOptions);
+    await this.selectAllCheckBoxs(fillValues.profession, this.professionOptions);
+    await this.selectAllCheckBoxs(fillValues.tools, this.seleniumTools);
+    await this.selectContinent(fillValues.continent);
+    await this.selectSeleniumComands(fillValues.commands);
+    await this.button.click();
+  }
+
+  public async manageAlert(): Promise<void> {
+    await browser.wait(ExpectedConditions.alertIsPresent(), 10000);
+    const alert = await browser.switchTo().alert();
+    await alert.accept();
+  }
+
+  public async getTitle(): Promise<string> {
+    await browser.sleep(3000);
```

Comment:
I know that using browser.sleep is not a very good idea, but without it tests are inconsistent because it can happen that the alert is still present at this point, and there is not ExpectedCondition that waits that an alert is NOT present or the "title is visible" as far as I researched. 
There is one that is ExpectedCondition.urlToBe, but It doesn't apply here because the URL doesn't changes when the form is completed.

Reply:
The problem is not here, the problems are the commands, for any reason that I don't know that component reacts with delay, so, the alert is displayed first. So please remove the sleep, and in the `fillForm` method, please change the order, and fill the commands first than the continent, that would bee enough

Reply:
That solved the problem :D